model/cluster_installation.go

```diff
@@ -33,6 +35,16 @@ type ClusterInstallationFilter struct {
 	IsActive       *bool
 }
 
+type ClusterInstallationStatus struct {
+	InstallationFound bool   `json:"installation_found"`
+	Replicas          *int32 `json:"replicas,omitempty"`
+	TotalPod          *int32 `json:"total_pod,omitempty"`
+	RunningPod        *int32 `json:"running_pod,omitempty"`
+	ReadyPod          *int32 `json:"ready_pod,omitempty"`
+	StartedPod        *int32 `json:"started_pod,omitempty"`
+	ReadyLocalServer  *int32 `json:"ready_local_server,omitempty"`
+}
```

Comment:
Historically, we have used the non-underline JSON naming scheme for our various structs: https://github.com/mattermost/mattermost-cloud/blob/master/model/group_status.go#L12-L20

This wasn't really an intentional decision, but sort of became the standard for the codebase. Do you think we should change that going forward or keep the original style. @fmartingr have any thoughts as well?

Reply:
Let's stick with the initial design. If we want to change, we also need to consider how to handle DB objects.

Reply:
My personal preference is `snake_case` for APIs, but I'd rather stick with `CamelCase` in the provisioner than switch everything to snakes, I'd rather be homogeneus than picky about this. 1/5